frontend/src/features/modules/decls/VerbPanel.tsx

```diff
@@ -0,0 +1,67 @@
+import { useNavigate } from 'react-router-dom'
+import { Badge } from '../../../components/Badge'
+import type { Ref, Type, Verb } from '../../../protos/xyz/block/ftl/v1/schema/schema_pb'
+
+const DataRef = ({ heading, r }: { heading: string; r: Ref }) => {
+  const navigate = useNavigate()
+  return (
+    <div
+      className={ioBlockClassName}
+      onClick={() => {
+        navigate(`/modules/${r.module}/data/${r.name}${window.location.search}`)
+      }}
+    >
+      <div className='text-sm'>{heading}</div>
+      <span className='text-xs'>
+        {r.module}.{r.name}
+      </span>
+    </div>
+  )
+}
+
+const ioBlockClassName =
```

Comment:
what is `ioBlock`? 

Ahh I see now, it's for showing request or response info stuff.

Reply:
ah yeah - I couldn't come up with a better name that wasn't disgustingly long. Definitely open to suggestions if you come up with anything 😂

Reply:
yeah it's a tricky one! we can just leave it for now since we'll probably follow up with verb/decl pages next.

Reply:
perfect